eng/common/pipelines/templates/steps/create-apireview.yml

```diff
@@ -6,18 +6,22 @@ parameters:
   GenerateApiReviewForManualOnly: false
   ArtifactName: 'packages'
   PackageName: ''
+  SourceRootPath: $(Build.SourcesDirectory)
+  WorkingDirectory: $(Pipeline.Workspace)
 
 steps:
   # ideally this should be done as initial step of a job in caller template
   # We can remove this step later once it is added in caller
   - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+    parameters:
+      WorkingDirectory: ${{parameters.SourceRootPath}}
```

Comment:
Why wouldn't this be WorkingDirectory? 

What context in Java has these not working?

Reply:
Java release job checks outs the code to `$(Pipeline.Workspace)/azure-sdk-for-java` and default working directory has `azure-sdk-build-tools` and causes script to fail with error `Invalid file path 'D:\a\_work\1\azure-sdk-build-tools\eng\common\scripts\Create-APIReview.ps1'`

Reply:
Which pipeline does this? I don't see multiple check out at https://github.com/Azure/azure-sdk-for-java/blob/e7deb7b872465c98a48089def44ebaa28f2d1430/eng/pipelines/templates/jobs/ci.yml#L219

Do you have a Pthat shows how this will be consumed?